### PR TITLE
[EVM] Add integration test for cw20 pointer 

### PR DESCRIPTION
**Pull request by stevenlanders**
## Describe your changes and provide context
- adds integration test for cw20
  - Deploys ERC20
  - Deploys CW20 Pointer to that ERC20
  - Calls query functions 
- renames tests to be clearer 
- dynamically invokes inside docker if it's running, otherwise runs locally

## Testing performed to validate your change
- test added to PR

### CODE REVIEW

File: contracts/test/CW20toERC20PointerTest.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        try {
+            await queryWasm(addr, operation, args);
+            // If the promise resolves, force the test to fail
+            expect.fail(`Expected rejection: address=${addr} operation=${operation} args=${JSON.stringify(args)}`);
+        } catch (error) {
+            expect(error.message).to.include("ERC20 does not support");
+        }
+    }
+
+    describe("query", function(){
```
Comment (jewei1997) [inline contracts/test/CW20toERC20PointerTest.js:46]: should we also test write functions?

File: contracts/test/lib.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                command = command.replace(/\.\.\//g, "/sei-protocol/sei-chain/");
+                const dockerCommand = `docker exec sei-node-0 /bin/bash -c 'export PATH=$PATH:/root/go/bin:/root/.foundry/bin && printf "12345678\\n" | ${command}'`;
+                execCommand(dockerCommand, resolve, reject);
+            } else {
+                // The container is not running, execute command normally
+                execCommand(command, resolve, reject);
+            }
+        });
+    });
+}
```
Comment (jewei1997) [inline contracts/test/lib.js:135]: nice

**Review by jewei1997 — APPROVED** [review]
overall LGTM, might be good to test a write function for CW20->ERC20.

File: contracts/test/CW20toERC20PointerTest.js (hunk 1 above)
Comment (stevenlanders) [reply contracts/test/CW20toERC20PointerTest.js:46]: added `transfer` which seems to work, though found an issue with approval - thinking we should fix in separate PR

**Pull request merged**